backends/tests/CMakeLists.txt

```diff
@@ -1,49 +1,40 @@
-string(COMPARE EQUAL "${GTEST_INCLUDE_DIR}" "" no_gtest_incl_dir)
-string(COMPARE EQUAL "${GTEST_LIB_DIR}" "" no_gtest_lib_dir)
-
-if(${no_gtest_incl_dir} OR ${no_gtest_lib_dir})
-    message(WARNING
-            "GTest is needed to test PyDPPL's backend test cases.  Pass in \
-            -DGTEST_INCLUDE_DIR and -DGTEST_LIB_DIR when you configure Cmake if\
-             you wish to run PyDPPL backend tests."
-    )
-else()
-  # We need thread support for gtest
-  find_package(Threads REQUIRED)
-
-  set(CMAKE_CTEST_COMMAND ctest -V)
-
-  # Emulate autotools like make check target to build tests
-  add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND})
-  enable_testing()
-
-  include_directories(${GTEST_INCLUDE_DIR})
-  include_directories(../include)
-
-  link_directories(${GTEST_LIB_DIR})
-
-  set(PYDPPL_BACKEND_TEST_CASES
-    test_sycl_kernel_interface
-    test_sycl_platform_interface
-    test_sycl_program_interface
-    test_sycl_queue_interface
-    test_sycl_queue_manager
+find_package(GTest REQUIRED)
```

Comment:
Same question. I was not sure if the macro is smart enough to look under conda prefix.

Reply:
I have tested on Linux. It works fine.

Reply:
On my system I have `gtest` installed in my conda environment and `find_package` fails to locate Gtest. What do I need to do to fix? I can install `gtest` through `apt`, but that is not what we want right?

Reply:
Ok. So to get it to work, I need `cmake` to come from conda as well. So it works. Nice!